dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt

```diff
@@ -4,35 +4,35 @@
 
 This guide will go over abilities, weapons & gear, approaching a boss, and refining mistakes.
 
-**Navigating: **To navigate this channel quickly either use the Table of Contents found in the pinned messages or quickly scroll through the guide to the desired section title.
+**Navigating: **To navigate this channel quickly either use the Table of Contents found below.
 
 .
 .img:https://img.pvme.io/images/xibEOmJ.png
 .
-> __**Ability Themes**__
+> __**Themes**__
 .tag:abilitythemes
 .
-Melee abilities vary in their themes. Some are strong individually, some comboed and some provide powerful utility values. Currently, they can be interpreted into four different classes with three different damage types.
+Melee abilities vary in their themes. Some are strong individually, some comboed and some provide powerful utility values. Currently, they can be interpreted into four classes with three damage types. Some abilities can have multiple classes.
 
 **__Classes: __**
-⬥ **Power**: Deal high damage.
-⬥ **Auxiliary**: Deal equal or less damage but enhance other abilities.
-⬥ **Utility**: provide low or non-damaging utility buffs.
-⬥ **Subpar**: as we know of lack any serious use. Generally avoid unless theorycrafted in advance.
+⬥ **Power**: High damage.
+⬥ **Helper**: Medium damage; enhance the next ability.
+⬥ **Utility**: Provides a buff over time or does other utility effects.
+⬥ **Subpar**: Lacks any serious use. Generally avoid unless theorycrafted.
 
-**__Damage Types:__**
-⬥ **Instant**: Deal damage immediately
+**__Damage Type:__**
+⬥ **Instant**: Immediate damage
     • *Examples:* <:cleave:535532878616985610> <:cane:535532878969438210> <:deci:535532879325822986> <:sever:535532879577612298> <:havoc:535532879300526080> <:gbarge:535532879250456578> <:punish:535532879439069184> <:slice:535532879594258432>
```

Comment:
Think this format can be condensed into a single line, because you're not really listing a few examples, but rather all of them. So personally I'd write it as something like maybe 

**Instant (deals damage immediately)** - <:cleave:535532878616985610> <:cane:535532878969438210> <:deci:535532879325822986> <:sever:535532879577612298> <:havoc:535532879300526080> <:gbarge:535532879250456578> <:punish:535532879439069184> <:slice:535532879594258432>